src/resources/restore_stage_bi.sh

```diff
@@ -474,7 +467,7 @@ command_runner "${CMD}" 11
 log "Environment:"
 export DLPX_LIBRARY_SOURCE=""
 export STAGINGPASS=""
-env | sort  >>$DEBUG_LOG
+env | grep -v 'STAGINGPASS' | sort >>$DEBUG_LOG
```

Comment:
The issue is the SOURCEPASS being shown in logs with env... maybe we can remove all occurrences of PASS from the command, like so 
`env | grep -v 'STAGINGPASS' | sort >>$DEBUG_LOG`